### DISCUSSION TITLE
IPKISS Compatibility

### DISCUSSION THREAD

**Discussion by z-y-n** [HelgeGehring/gdshelpers#22] [General]
Hi! Very excited to see this work. Our team is currently using IPKISS for layout related stuffs and has a code base already building around IPKISS. I'm wondering, if we were to migrate things over to this tool you are building, how easy do you think the process would be?

**Comment by HelgeGehring** [comment]
Hey zeyuen, nice to hear that you like gdshelpers!
I never used ipkiss, but I just asked someone from our group who switched from ipkiss to gdshelpers as he joined our group. He told me switching was quite easy for him :)

**Comment by z-y-n** [comment]
Thanks for the response, that's great to hear Helge. I'm just wondering
what the future plan/roadmap is for the project, and if you guys are open
to feature requests?

On Wed, Aug 26, 2020 at 8:55 AM Helge Gehring <notifications@github.com>
wrote:

> Hey zeyuen, nice to hear that you like gdshelpers!
> I never used ipkiss, but I just asked someone from our group who switched
> from ipkiss to gdshelpers as he joined our group. He told me switching was
> quite easy for him :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HelgeGehring/gdshelpers/discussions/22#discussioncomment-57081>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALP52MH46C5DJSS3AC6NYZTSCUA35ANCNFSM4QLCCPMA>
> .
>

-- 
xanadu.ai <http://xanadu.ai>

"To build quantum computers that are useful 
and available to people everywhere"



This e-mail message is privileged, 
confidential and subject to
copyright. Any unauthorized use or disclosure 
is prohibited.
Le contenu du présent courriel est privilégié, confidentiel 
et
soumis à des droits d'auteur. Il est interdit de l'utiliser ou
de le 
divulguer sans autorisation.

**Comment by HelgeGehring** [comment]
At the moment, most updates come from problems we have while designing layouts. Besides that, I'm planning to add an integration of Meep for directly simulating designed parts (see #9 ).
We're very open to feature requests, bug reports and pull requests :)
What features would you need?
